### DISCUSSION TITLE
Doubts regarding this repository - Ask Here

### DISCUSSION THREAD

**Discussion by Sanskriti1102** [Sanskriti1102/Techtalksintelligenz#11] [Q&A] [answered]
**Ask any doubts related to the repository here!**

**Comment by Rahul7raj** [comment, marked as answer]
@Sanskriti1102 the `package.json` file is not available in the repo. please update it.

**Reply by Sanskriti1102** [reply]
@Rahul7raj  since the website is html css mostly so package isn't  there !
